### PR TITLE
feat(TDP-4032): home headerbar react

### PR DESCRIPTION
**Pull request by charlesnguyen**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-4032

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [x] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

**(Optional) What is the current behavior?**
(Additional information to the Jira)

**(Optional) What is the new behavior?**
(Additional information to the Jira)

**(Optional) Other information**:

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/407/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/407/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/409/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/409/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/410/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/410/)

File: dataprep-webapp/src/app/components/widgets-containers/breadcrumb/breadcrumb-container.spec.js [hunk 1]
```diff
@@ -99,9 +99,9 @@ describe('Breadcrumb container', () => {
 
 			// then
 			const parentFolder = element.find('.breadcrumb > li').eq(0);
-			expect(parentFolder.find('button').eq(0).text()).toBe('…');
+			expect(parentFolder.find('button').eq(0).text()).toBe('...');
```
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/widgets-containers/breadcrumb/breadcrumb-container.spec.js]: I just report https://github.com/Talend/data-prep/commit/110e7e8a47c21db177d794b5c5aad597185e543e

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/416/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/416/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/421/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/421/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/426/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/426/)

**Pull request merged**